modules/oe_content_publication/oe_content_publication.post_update.php

```diff
@@ -41,7 +41,14 @@ function oe_content_publication_post_update_00003() {
   if (!\Drupal::service('module_handler')->moduleExists('oe_content_organisation_reference')) {
     $module_installer->install(['oe_content_organisation_reference']);
   }
-  $module_installer->install(['path', 'inline_entity_form']);
+  $module_installer->install([
+    'composite_reference',
```

Comment:
Not all modules are enabled that have been added as dependencies, for example oe_media_avportal, oe_content_reference_code_field etc.